include/openpose_plus/operator/dnn/tensorrt.hpp

```diff
@@ -0,0 +1,118 @@
+#pragma once
+
+/// \file tensorrt.hpp
+/// \brief The DNN engine implementation of TensorRT.
+/// \author Jiawei Liu(github.com/ganler)
+
+#include "../../utility/model.hpp"
+#include <NvInferRuntime.h>
+#include <NvInferRuntimeCommon.h>
+#include <future>
+
+#include <ttl/cuda_tensor>
+
+#include "../../utility/data.hpp"
+
+namespace poseplus {
+
+/// Element data type.
+using data_type = nvinfer1::DataType;
```

Comment:
alias is not used

Reply:
Yes. But I don't really like users to touch types from other libraries(e.g., `nvinferN::XXX`). 
And it's meaningful when we support other engines. What do you think?